com.zoho.Notebook.metainfo.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This file is not referenced and installed in the manifest.

Reply:
Ops! I didn't realize that. Referenced now.